app/views/layouts/application.html.haml

```diff
@@ -10,6 +10,16 @@
     %header
       #headerbar
         = link_to "Social Wiki", "/", {id: 'logo'}
-    
+        
+        - if current_user
+          = label_tag current_user.username
+          = link_to 'Logout', logout_path, {id: 'logo'}
+  
```

Comment:
I find this extra line quite confusing. Also there are a lot of extra lines in this file. Sorry for being nitpicky.